cognite/v05/timeseries.py

```diff
@@ -288,12 +288,18 @@ def post_multi_tag_datapoints(timeseries_with_datapoints: List[TimeseriesWithDat
 
         project (str): Project name.
 
+        use_gzip (bool): Whether or not to gzip the request
+
     Returns:
         An empty response.
     """
     api_key, project = config.get_config_variables(kwargs.get("api_key"), kwargs.get("project"))
     url = config.get_base_url(api_version=0.4) + "/projects/{}/timeseries/data".format(project)
 
+    use_gzip: bool = False
+    if "use_gzip" in kwargs:
+        use_gzip = kwargs.get("use_gzip")
```

Comment:
Nitpick: Du kan skrive `use_gzip: bool = kwargs.get("use_gzip", False)` for å defaulte til False dersom den ikke eksisterer i kwargs.